File: trabalho 1/programas tarefa 1/java/micro/micro03.java
public class micro03{
    public static void main(String[] args){
        int numero;
        System.out.print("Digite um numero: ");
        numero = Integer.parseInt(System.console().readLine());
        if(numero >= 100){
            if(numero <= 200){
                System.out.println("O numero esta no intervalo entre 100 e 200");
            }else{
                System.out.println("O numero nao esta no intervalo entre 100 e 200");
            }
        }else{
            System.out.println("O numero nao esta no intervalo entre 100 e 200");
        }
    }
}

File: trabalho 6/codigos/lua/micro/micro03.lua
function main() int
    int numero
    print("Digite um numero: ")
    numero = io.read()
    if numero >= 100 then
        if numero <= 200 then
            print("O numero esta no intervalo entre 100 e 200\n")
        else
            print("O numero nao esta no intervalo entre 100 e 200\n")
        end
    else
        print("O numero nao esta no intervalo entre 100 e 200\n")
    end
    return 1
end

main()
